definitions/divorce/json/CaseEvent/CaseEvent-deemed-and-dispensed-nonprod.json

```diff
@@ -213,7 +213,7 @@
     "CaseTypeID": "DIVORCE",
     "ID": "IssueAOSOffline_Respondent_FromServiceAppRejected",
     "Name": "Issue AOS pack to respondent",
-    "Description": "Answer received",
+    "Description": "Issue AOS pack to respondent",
```

Comment:
Here should be "Issue AOS (offline) for respondent from ServiceApplicationNotApproved" to follow convention

Reply:
The ACs state:
"Name": "Issue AOS pack to respondent", 
"Description": "Issue AOS pack to respondent",